ports/psoc6/main.c

```diff
@@ -178,10 +164,6 @@ void mpy_task(void *arg) {
     // ANSI ESC sequence for clear screen. Refer to  https://stackoverflow.com/questions/517970/how-to-clear-the-interpreter-console
     mp_printf(&mp_plat_print, "\033[H\033[2J");
 
-    mp_printf(&mp_plat_print, MICROPY_BANNER_NAME_AND_VERSION);
-    mp_printf(&mp_plat_print, "; " MICROPY_BANNER_MACHINE);
-    mp_printf(&mp_plat_print, "\nUse Ctrl-D to exit, Ctrl-E for paste mode\n");
```

Comment:
This is good to have no?

Reply:
Yes but it looks a bit weird written before the banner... 
So you can find that in the help()? 
And the banner is rewritten any the repl_friendly...